content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -77,58 +77,62 @@ Deletion removes both the `VolumeSnapshotContent` object from the Kubernetes API
 
 ## Volume Snapshot Contents
 
-Each VolumeSnapshotContent contains a spec, which is the specification of the volume snapshot.
+Each VolumeSnapshotContent contains a spec and status, which is the specification and status of the volume snapshot content. In dynamic provisioning, the `VolumeSnapshotContent` object is created by the snapshot common controller.
```

Comment:
2nd half of this sentence is sort of redundant...

"In the pre-provisioned snapshot case, the object is created by an administrator"

Reply:
Done